package.json

```diff
@@ -50,12 +50,12 @@
     "css-loader": "^0.28.1",
```

Comment:
New feature should get a new version. v 1.6.0 (please also update in the `package.xml` and `package-lock.json`

Reply:
Done